src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -44,18 +44,21 @@ class ElasticIsotropicUpdates : public SolidBaseUpdates
    * @brief Constructor
    * @param[in] bulkModulus  The ArrayView holding the bulk modulus data for each element.
    * @param[in] shearModulus The ArrayView holding the shear modulus data for each element.
+   * @param[in] thermalExpansionCoefficient The ArrayView holding the thermal expansion coefficient data for each element.
    * @param[in] newStress    The ArrayView holding the new stress data for each quadrature point.
    * @param[in] oldStress    The ArrayView holding the old stress data for each quadrature point.
    * @param[in] disableInelasticity Flag to disable plasticity for inelastic models
    */
   ElasticIsotropicUpdates( arrayView1d< real64 const > const & bulkModulus,
                            arrayView1d< real64 const > const & shearModulus,
+                           arrayView1d< real64 const > const & thermalExpansionCoefficient,
                            arrayView3d< real64, solid::STRESS_USD > const & newStress,
                            arrayView3d< real64, solid::STRESS_USD > const & oldStress,
                            const bool & disableInelasticity ):
     SolidBaseUpdates( newStress, oldStress, disableInelasticity ),
     m_bulkModulus( bulkModulus ),
-    m_shearModulus( shearModulus )
+    m_shearModulus( shearModulus ),
+    m_thermalExpansionCoefficient( thermalExpansionCoefficient )
```

Comment:
Addition of `thermalExpansionCoefficient` should also be applied to [ElasticOrthotropic.hpp](https://github.com/GEOSX/GEOSX/blob/develop/src/coreComponents/constitutive/solid/ElasticOrthotropic.hpp) and 
[ElasticTransverseIsotropic.hpp](https://github.com/GEOSX/GEOSX/blob/develop/src/coreComponents/constitutive/solid/ElasticTransverseIsotropic.hpp)? or it will be handled in future work?

Reply:
> Thank @francoishamon for implementing this PR, I will test and validate it on several examples with existing reference results.

@sytuannguyen This and future PRs on THM will cause some conflicts to the viscoplasticity PR(#2004). And the tutorial examples for triaxialDriver are not covered by integratedTests and should also be updated. Multiple smokes examples for thermo-mechanical (elastic, plastic, viscoplastic) and poro-thermo-mechanical model should be added to the integratedTests.

Reply:
Done, I added the `thermalExpansionCoefficient` to the `ElasticOrthotropic` and `ElasticTransverseIsotropic` models.